src/org/zaproxy/zap/extension/jruby/resources/help_ar_SA/JavaHelpSearch/DOCS

```diff
@@ -0,0 +1 @@
+�����������������
```

Comment:
You can use interactive rebase to edit/amend the first commit to remove that file:
https://help.github.com/articles/using-git-rebase/